test/plugins/test_thumbnails.py

```diff
@@ -100,12 +100,10 @@ def exists(path):
 
     @patch("beetsplug.thumbnails.ThumbnailsPlugin._check_local_ok", Mock())
     @patch("beetsplug.thumbnails.ArtResizer")
-    @patch("beetsplug.thumbnails.util")
+    @patch("beets.util.syspath", Mock(side_effect=lambda x: x))
```

Comment:
Oh, wow, that's a test with a lot of mocks in place. I'm wondering whether that can't be done in a more robust way, relying less on the implementation. Anyway, that's for a different PR.

Reply:
Yeah, that's pretty much useless and overly fragile implementation tests that will break for any implementation adjustment 🤷🏼‍♂️